test/stripe/customer_test.rb

```diff
@@ -57,7 +57,7 @@ class CustomerTest < Test::Unit::TestCase
       should "delete a discount" do
         customer = Stripe::Customer.retrieve("cus_123")
         customer = customer.delete_discount
-        assert_requested :delete, "#{Stripe.api_base}/v1/customers/#{customer.id}/discount"
+        assert_requested :delete, "#{Stripe.api_base}/v1/customers/cus_123/discount"
```

Comment:
Had to change this test because stripe-ruby mutates the customer in place and changes its id as stripe-mock doesn't know this is an endpoint with the customer id in it